updating/updating-cluster-rhel-compute.adoc

```diff
@@ -18,6 +18,7 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
 * Support for {op-system-base}7 workers is removed in {product-title} {product-version}. You must replace {op-system-base}7 workers with {op-system-base}8 or {op-system} workers before upgrading to {product-title} {product-version}. Red Hat does not support in-place {op-system-base}7 to {op-system-base}8 upgrades for {op-system-base} workers; those hosts must be replaced with a clean operating system install.
 * If your cluster uses manually maintained credentials, ensure that the Cloud Credential Operator (CCO) is in an upgradeable state. For more information, see _Upgrading clusters with manually maintained credentials_ for xref:../installing/installing_aws/manually-creating-iam.adoc#manually-maintained-credentials-upgrade_manually-creating-iam-aws[AWS], xref:../installing/installing_azure/manually-creating-iam-azure.adoc#manually-maintained-credentials-upgrade_manually-creating-iam-azure[Azure], or xref:../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-maintained-credentials-upgrade_manually-creating-iam-gcp[GCP].
 * If your cluster uses manually maintained credentials with the AWS Secure Token Service (STS), obtain a copy of the `ccoctl` utility from the release image being updated to and use it to process any updated credentials. For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-upgrading[_Upgrading an OpenShift Container Platform cluster configured for manual mode with STS_].
+* If you run an Operator or you have configured any application with the Pod Disruption Budget, you might experience a block during the upgrade process. The `PodDisruptionBudget` with `minAvailable: 1`  may block the eviction process while nodes are drained to apply pending MachineConfigs. If several nodes are rebooted, all the pods may run on only one node, and the `PodDisruptionBudget` can prevent the node drain.
```

Comment:
How is this related to the architecture diagram?

Reply:
@zanetworker I fixed it. sorry, the irrelevant commit seemed to be added in this PR.